cpp/include/cudf/rolling.hpp

```diff
@@ -379,6 +381,119 @@ std::unique_ptr<column> grouped_time_range_rolling_window(
   std::unique_ptr<aggregation> const& aggr,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief  Applies a grouping-aware, value range-based rolling window function to the values in a
+ *         column.
+ *
+ * This is a generalization of `grouped_time_range_rolling_window()` to work with non-timestamp
+ * columns. Like `grouped_time_range_rolling_window()`, this function aggregates values in a window
+ * around each element of a specified `input` column. It differs from
+ * `grouped_time_range_rolling_window()` in that instead of a timestamp column, the ordered column
+ * can be of integral types as well as timestamps.
```

Comment:
Documentation reads easier if it doesn't require knowledge of how another API works, i.e., it should stand on its own.

Reply:
Agreed. This is of particular importance considering that the new API obviates `grouped_time_range_rolling_window()`, and should eventually replace it.
I will reword this.

Reply:
Cleaned up the documentation here. Removed references to `grouped_time_range_rolling_window()`.